third_party/protobuf/BUILD

```diff
@@ -15,6 +15,20 @@ genrule(
     outs = [
         "bin/protoc",
 
+        "include/google/protobuf/arena.h",
```

Comment:
How were you able to determine the long list of required includes here? Could you provide a link in a comment?